website/content/partials/components/platform-exec.mdx

```diff
@@ -29,7 +29,7 @@ template. The top of the documentation there has details on the format.
 
 The following template values are always available:
 
-- ".Env" (map<string>string) - These are environment variables that should
+- ".Env" (map<string\>string) - These are environment variables that should
```

Comment:
This already needed to be escaped, but was silently failing (see [live](https://www.waypointproject.io/plugins/exec#common-values), the text `<string>` does not appear because it is interpreted as an opening HTML tag).

Now that we're processing partials with MDX, this issue no longer fails silently. Note that all other uses of similar tag-like text I could find (eg `<string>`) are rendered as inline code, or in code blocks, which is why this is typically not an issue.